utilities/repeder/repeder.cc

```diff
@@ -259,6 +259,10 @@ int main (int nargs, char ** args)
   //! Input quality cut results. -MK added 11-02-2022
   std::ifstream qualFile(qual_file);
   if (qual_file) {
```

Comment:
So I guess in the case where qual_file is 0 (or nullptr), this will attempt to open that and fail, but I guess it probably doesn't really matter.

Reply:
Indeed, It is not neat. (let me do minor modification...:))